src/main/java/com/google/cloud/spanner/pgadapter/ConnectionHandler.java

```diff
@@ -314,10 +315,23 @@ public ConnectionMetadata getConnectionMetadata() {
     return connectionMetadata;
   }
 
+  public synchronized IntermediateStatement getActiveStatement() {
+    return activeStatementsMap.get(this.connectionId);
+  }
+
+  public synchronized ConnectionStatus getStatus() {
```

Comment:
these aren't being used. are they for a future PR?

Reply:
This was used in a later change I added.  I'll remove it from this PR.